src/shared/lib/StoreUtils.ts

```diff
@@ -252,6 +252,24 @@ export async function fetchCopyNumberSegments(studyId:string,
     }
 }
 
+export async function fetchCopyNumberSegmentsForSamples(samples: Sample[],
```

Comment:
I'm not sure it make sense to hide the promise inside the utility function in this way.  If we're going to move the fetch to a utility then it should return promise explicitly (typing).   The importance of doing this is 1. if there's logic we really need to test in the handling of the response, 2. If the logic is lengthy and bloats the store.   Neither of those conditions is true here so I don't see the point.

Reply:
Also, in the case of #1, it's the sychronous logic of handling the response that we really want to decouple for the purpose of testing.